### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Assorted spelling fixes

### CODE REVIEW

File: internal/runtimeconfig/runtime_config.go [hunk 1]
```diff
@@ -28,7 +28,7 @@
 //	    type: "redis"
 //	    options:
 //	      - name: url
-//	        value: "redis://localghost:9000"
+//	        value: "redis://localhost:9000"
```
Comment (jsoref) [inline internal/runtimeconfig/runtime_config.go:31]: I'm not certain about this. It's possible this is being cute and refers to a computer that is _not_ `localhost`. But I didn't see anything in this repository that indicated it liked being "cute"...

Comment (endocrimes) [reply internal/runtimeconfig/runtime_config.go:31]: This _was_ deliberate 😅  (partially to indicate that this isn't "real" config)

Comment (jsoref) [reply internal/runtimeconfig/runtime_config.go:31]: Might I suggest using `something` instead?

Comment (endocrimes) [reply internal/runtimeconfig/runtime_config.go:31]: i'd rather keep it fun, but don't really care (the rest of the changes lgtm fwiw)

File: internal/controller/spinapp_controller.go [hunk 2]
```diff
@@ -248,7 +248,7 @@ func (r *SpinAppReconciler) ensureCASecret(ctx context.Context, caSecretName, na
 	}
 
 	err := r.Client.Get(ctx, types.NamespacedName{Name: caSecretName, Namespace: namespace}, secret)
-	if !apierrors.IsNotFound(err) { // secret is not not found
+	if !apierrors.IsNotFound(err) { // secret is not found
```
Comment (michelleN) [inline internal/controller/spinapp_controller.go:251]: I think the additional "not" originally here is on purpose. I think this comment is confusing though and we should get rid of it entirely.

**Review by michelleN — COMMENTED** [review]
Nice catches! I just left one comment above.

**Review by endocrimes — APPROVED** [review]

**Review by calebschoepp — APPROVED** [review]
Thanks for the contribution. I would say we're not to fussed about the shape of the commits. Feel free to sign them all or squash it to one commit (I might have a slight but totally non-blocking preference for this).

**Review by endocrimes — APPROVED** [review]

**Pull request merged**